content/index/index.html

```diff
@@ -21,6 +21,7 @@ <h2>About the Index</h2>
     <ul>
       <li><a href="#examples_by_role_label">Examples by Role</a></li>
       <li><a href="#examples_by_props_label">Examples by Properties and States</a></li>
+      
```

Comment:
Seems that the `cheerio.remove()` [leaves a blank line behind](https://github.com/cheeriojs/cheerio/issues/86). Reviewers can let me know if I should find an alternate solution for removal.